acceleration/README.md

```diff
@@ -0,0 +1,168 @@
+# Sample
```

Comment:
It was not obvious to me in my first pass that this sample chains Spice instances together - that needs an explanation and it would be good to have a figma to illustrate the architecture here, to make it easier to understand.

Reply:
Please resolve / address in next iteration.  Closing comment to merge.